README.md

```diff
@@ -128,7 +128,6 @@ You can deploy this project using:
 
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you fix the broken `Live Demo` link in your README file to make it more professional as it shows a **404** error page once clicked. As you don;t need to deploy your project online, I suggest you completely delete it from your README.